openassessment/xblock/static/js/src/lms/oa_peer.js

```diff
@@ -123,9 +152,14 @@ OpenAssessment.PeerView.prototype = {
                 // Override default form submission
                 eventObject.preventDefault();
 
+
+                //Status will change in update announce it to the Screen Reader after Render
+                view.announceStatusChange = true;
```

Comment:
I'm not clear why this explicit call is necessary (as opposed to the handling above in the render callbacks.